java/memory/memory-netty/src/main/java/io/netty/buffer/PooledByteBufAllocatorL.java

```diff
@@ -265,7 +265,7 @@ public MemoryStatusThread() {
       @Override
       public void run() {
         while (true) {
-          memoryLogger.trace("Memory Usage: \n{}", PooledByteBufAllocatorL.this.toString());
+          memoryLogger.trace("Memory Usage: \n{}", PooledByteBufAllocatorL.this.allocator);
```

Comment:
Since this is a background thread, doesn't the test need to account for a potential race condition?

Reply:
> Since this is a background thread, doesn't the test need to account for a potential race condition?

Oh, please could you help me how/when this method execute/run? Or Could you help me with an example about the current race condition problem? This will help me to understand in more detail how this run behind the scene.

Reply:
Ok let's continue as it is now.